0-raw_data_processing/0-240409-E-MTAB-12916/0-240409-kallisto_bustools/0-240409-mapping_kallisto_bustools_E-MTAB-12916.ipynb

```diff
@@ -0,0 +1,1989 @@
+{
```

Comment:
<p>This loop is nice, but I think we could make it better. </p><p> </p><ul><li>Personally, I would declare the individual files together with their paths at the beginning. This makes the command cleaner. </li><li>We also want to infer trajectories from these data, so I will also add the RNA velocity workflow to <code>kb count</code>, I think is  <code>--workflow lamanno</code> now, but I will double check. </li><li>Also, on this topic, I believe that some of these were not single cell but single nuclei, which may be important to pass along as well. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/human_heart_mapping/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>i changed the workflow to lamanno for kb ref and now the process takes much longer (an hour and still going), is that normal?</p>

Reply:
Can you share the error?